src/common/consumers/email.consumer.ts

```diff
@@ -0,0 +1,27 @@
+import { Job } from 'bull';
+import { Process, Processor } from '@nestjs/bull';
+import { MailerService } from '@nestjs-modules/mailer';
+import { ConfigService } from '@nestjs/config';
+import { EmojiLogger } from 'src/common/logger/EmojiLogger';
+
+@Processor('email-sms')
+export class EmailConsumer {
+  private readonly logger = new EmojiLogger();
+  constructor(
+    private readonly mailService: MailerService,
+    private configService: ConfigService,
+  ) {}
+  @Process('email')
+  async sendEmail(job: Job) {
+    await this.mailService
+      .sendMail({
+        from: 'naomi.hackett60@ethereal.email ',
+        to: job.data.email,
+        subject: 'Invitation',
+        html: `<p>Please verify your email by clicking the following link: <a href="http://localhost:6000/verify?code=${job.data.code}">Verify Email</a></p>`,
```

Comment:
Application URL must be also taken from the env variables. Because if you deploy this app to the real domain, you nee to change the localhost to the real domain name directly in the code.